### ISSUE TITLE
var-spacing: false-positive on {% raw %} '{{.Count}}' {% endraw %}

### ISSUE THREAD

**Issue by ktomk** [ansible/ansible-lint#1685]
(Draft Report)

Error:

~~~
var-spacing: Variables should have spaces before and after: echo {% raw %} '{{.Count}}' {% endraw %}
~~~

Cause:

Escaping Jinja2 with `{% raw %} ... {% endraw %}` tags. [href](https://docs.ansible.com/ansible/latest/user_guide/playbooks_advanced_syntax.html#unsafe-or-raw-strings)

##### Issue Type

- Bug Report

##### Ansible and Ansible Lint details

```console (paste below)
ansible --version
ansible 2.10.7
  config file = /etc/ansible/ansible.cfg
  configured module search path = ['/home/mot/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3/dist-packages/ansible
  executable location = /usr/bin/ansible
  python version = 3.8.10 (default, Jun  2 2021, 10:49:15) [GCC 9.4.0]

ansible-lint --version
ansible-lint 5.1.2 using ansible 2.10.12
```

- ansible installation method: OS package
- ansible-lint installation method: pip (venv, incl. the ansible version in use by ansible-lint)

##### OS / ENVIRONMENT

Ubuntu 20.04 LTS

##### STEPS TO REPRODUCE

```console (paste below)

```

##### Desired Behaviour

Possible security bugs should be reported via email to `security@ansible.com`

##### Actual Behaviour

False positive for the mustaches in raw.

> `var-spacing: Variables should have spaces before and after: [...]`

[Ref Ansible Documentation](https://docs.ansible.com/ansible/latest/user_guide/playbooks_advanced_syntax.html#unsafe-or-raw-strings)

**Comment by ganeshrn** [comment]
Related to https://github.com/ansible-community/ansible-lint/issues/1702

**Comment by ktomk** [comment]
@ganeshrn: Can confirm same same but different only:  `!unsafe` in #1702 , `{% raw %} ... {% endraw %}` here.

I'm fine w/ extending the scope there to close here and/or/but also to fix there first and then bubble after here (this report has not so good reproduce information). Feel free to handle as it suits best.

Remark: For my subjective report the scenario are golang templates which allow to add spaces so that there is a workaround in my very own scenario. Just FYI, YMMV.
